tools/testbuild.sh

```diff
@@ -276,6 +281,18 @@ function build {
   return $fail
 }
 
+function run {
+  if [ ${RUN} -ne 0 ]; then
+    run_script="$path/run"
```

Comment:
should we add .sh suffix?

Reply:
nothing assumes it's a shell script.
but i can add .sh if you prefer.

Reply:
I am fine with either.